src/libstd/sys/windows/ext/process.rs

```diff
@@ -97,3 +97,32 @@ impl ExitStatusExt for process::ExitStatus {
         process::ExitStatus::from_inner(From::from(raw))
     }
 }
+
+/// Windows-specific extensions to the `std::process::Command` builder
+#[unstable(feature = "windows_process_extensions", issue = "37827")]
+pub trait CommandExt {
+    /// Sets the [process creation flags][1] to be passed to `CreateProcess`.
+    ///
+    /// These will always be ORed with `CREATE_UNICODE_ENVIRONMENT`.
+    /// [1]: https://msdn.microsoft.com/en-us/library/windows/desktop/ms684863(v=vs.85).aspx
+    #[unstable(feature = "windows_process_extensions", issue = "37827")]
+    fn set_creation_flags(&mut self, flags: u32) -> &mut process::Command;
+    /// Add `flags` to the the [process creation flags][1] to be passed to `CreateProcess`.
+    ///
+    /// These will always be ORed with `CREATE_UNICODE_ENVIRONMENT`.
+    /// [1]: https://msdn.microsoft.com/en-us/library/windows/desktop/ms684863(v=vs.85).aspx
+    #[unstable(feature = "windows_process_extensions", issue = "37827")]
+    fn add_creation_flags(&mut self, flags: u32) -> &mut process::Command;
```

Comment:
In terms of naming and functionality I think we'll want to follow the same pattern as [
OpenOptions`](https://github.com/rust-lang/rust/blob/ecff71a45c9d1aff9ba5dbc212cb1353578525ee/src/libstd/sys/windows/ext/fs.rs#L73-L92) for specifying Windows-specific features. In that sense perhaps we can call this `creation_flag` and avoid the `set`/`add` variants and have just the equivalent of `set`?